README.md

```diff
@@ -1,5 +1,7 @@
 # Serilog ApplicationServices Enrichment Library
 
+# -----------Deprecated-----------
```

Comment:
Hier zou ik ook nog wat bijkomende info bijzetten.

```
## -----------Deprecated-----------

This toolbox is no longer maintained because application information is automatically added to the logs on OpenShift/Kubernetes and the fields are not required according to the [Digipolis logging guidelines](https://github.com/digipolisantwerpdocumentation/logging-requirements).
```